### PR TITLE
[DCFS-2] Develop a Product Page using HTML

### PR DESCRIPTION
**Pull request by kemysola**
## Task: `Product page using html.`
- `removed the css properties as adviced.`
- 
Screen Shot 2021-02-17 at 12 09 38 PM

### CODE REVIEW

**Comment by kemysola** [comment]
Good afternoon ma,@adaugochi . Kindly review my new commit ma, I have effected the changes as adviced ma, thank you .

File: submissions/adekemiAwomolo/html/index.html [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+<br>
+
+<br>
+
+</aside>	
+<h2 class='divider'> Products</h2>
+
+<section>
+	<div class='imm'>
+	<img src='asset/it.jpg' height='350' width='690'>
```
Comment (ToluAkin) [inline submissions/adekemiAwomolo/html/index.html:33]: Remove width and height here @kemysola

**Review by ToluAkin — CHANGES_REQUESTED** [review]
There are still other scholars files in your pull request

File: submissions/adekemiAwomolo/html/index.html (hunk 1 above)
Comment (kemysola) [reply submissions/adekemiAwomolo/html/index.html]: Good afternoon ma @ToluAkin .I have removed the width and the height on the image as requested.Kindly review ma, thank you.

**Comment by ToluAkin** [comment]
> There are still other scholars files in your pull request 

@kemysola Not yet fixed. Check the files tabs to see all the files added in your pull request.

**Review by adaugochi — CHANGES_REQUESTED** [review]
You are deleting other people work that doesn't belong to you
image

**Comment by adaugochi** [comment]
They are part of your PR, kindly revert the changes

**Pull request closed without merging**